CMakeLists.txt

```diff
@@ -82,6 +83,10 @@ add_definitions(
     -DQT_MESSAGELOGCONTEXT #enable function name and line number in debug output
 )
 
+if(QUICK_COMPILER)
+    add_definitions(-DQUICK_COMPILER) # Enable use of QtQuick compiler
```

Comment:
@claucambra Aren't you missing `=1` after `-DQUICK_COMPILER`?

Reply:
The default state of the QUICK_COMPILER variable is handled by the `option(QUICK_COMPILER "Use QtQuick compiler to improve performance" OFF)`, here we are just adding the definition so we can do `cmake .. -DQUICK_COMPILER=TRUE`